docs/apps/interchain-accounts/client.md

```diff
@@ -0,0 +1,232 @@
+<!--
+order: 7
+-->
+
+# Client
+
+## CLI
```

Comment:
Moving the CLI documentation and/or adding documentation to it (unsure if that happened) in the future should be done in a separate pr to main. This goes for any logically isolated concept